packages/daemon/src/runner/job-runner.ts

```diff
@@ -244,7 +247,12 @@ export class JobRunner {
     const modelPath = path.join(this.opts.runnable.workingDir, 'model');
 
     await this.runDataCollect(dataDir, modelPath);
-    const dataset = await this.runDataAccess(dataDir);
+
+    // copy data to a tmp directory
+    await fs.ensureDir(dataDir);
+    await fs.copy(dataDir, this.tmpdataDir);
```

Comment:
```suggestion
    await fs.copy(dataDir, this.opts.runnable.dataDir);
```

It means we moving the pipeline data directory to a job's data directory.

Reply:
done